evaluation.md

```diff
@@ -6,8 +6,59 @@ _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+10. The syntax is not as clear as we wanted it to be initially. Now we're relying
+on one method, `If`, that calls every other method.  Thus, we're passing in strings as
+parameters and calling the other relevant functions, `Nothing`, `Something`, and `Go` from
+the method `If`. 
+
+We've changed from having 
+```
+North = {
+    if (Nothing[N]) Go North
+    if (Something[N] and Nothing [W]) Go West
+    if (Something[N, W]) Go East
+}
+``` 
+to represent all the rules for going North, to having 
+```
+If ("N", "Nothing(N)","Go(North)")
```

Comment:
This is interesting, by making large chunks of the program strings it is as though you essentially converted this into a parsing problem (like the external DSL).  We also had to make similar compromises, but we provided for implicit conversions from strings to in an attempt to get more modularity rather than having to parse an entire string and rely on regex error reporting to tell the user their syntax errors.